README.md

```diff
@@ -20,15 +20,15 @@ Supports message delete, update, lock command...
   - lock all - Lock all channels in the network: `c!lock all`
   - unlock - Unlock the current channel: `c!unlock`
   - unlock all - Unlock all channels: `c!unlock all`
-  - commands are usable by poeple with roles in managerRoles only.
+  - commands are usable by poeple with roles in manageRoles only.
```

Comment:
i think this is actually `managerRoles`

Reply:
Don't think you can manager a role :P

Reply:
![image](https://user-images.githubusercontent.com/24302436/89229616-a8830600-d5d9-11ea-8c20-e793f20d78b3.png)

Reply:
oh, this was meant to reference the `managerRoles` property in the config, sorry for the confusion. Maybe a rewording then since it doesn't seems clear

Reply:
```suggestion
  - commands are usable by poeple that have one of the roles in the `managerRoles` field in the config.
```